examples/with-react-intl/components/PageWithIntl.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react'
+import {IntlProvider, addLocaleData, injectIntl} from 'react-intl'
+
+// Register React Intl's locale data for the user's locale in the browser. This
+// locale data was added to the page by `pages/_document.js`. This only happens
+// once, on initial page load in the browser.
+if (typeof window !== 'undefined' && window.ReactIntlLocaleData) {
```

Comment:
@sedubois do you want to send a PR to my branch with your suggested changes? Then I can merge your commits into this branch/PR.